### ISSUE TITLE
The plugin has stopped working with the latest GitLab update

### ISSUE THREAD

**Issue by jeremymayhem** [wizpanda/gitlab-chrome#1]
GitLab version 16.0 breaks the plugin, probably because of the layout changes in the merge request view.  I'm amazed at how much I miss the capability of this plugin when it is gone.

**Comment by sagrawal31** [comment]
Hey @jeremymayhem, thanks for recognising the capability of this plugin. I did this for my own purpose. Let me know if you want to see this plugin working again.

**Comment by sagrawal31** [comment]
By the way, were you using the full width editor in MR view?

**Comment by jeremymayhem** [comment]
Yes, I was using the full width editor.  I would very much like to see this plugin working again!

Thanks!

**Comment by sagrawal31** [comment]
@jeremymayhem I stopped using the full-width editor from this extension way before because Gitlab already had the inbuilt support (or came later after I wrote the plugin). You can enable this in preferences
![image](https://github.com/wizpanda/gitlab-chrome/assets/1804514/960193f4-024d-4cbc-81b9-72f362ba4ca5)

Let me know if you are looking for something else.
